OpenOversight/app/utils.py

```diff
@@ -226,3 +226,10 @@ def compute_leaderboard_stats(select_top=25):
                             .order_by(func.count(Face.user_id).desc()) \
                             .limit(select_top).all()
     return top_sorters, top_taggers
+
```

Comment:
`flake8` is complaining about only having a single newline between this function and the previous one.